openlibrary/templates/type/edition/view.html

```diff
@@ -254,12 +254,12 @@ <h6>$label</h6>
         </div>
 
         $if page.description or (work and work.description):
-          $if work and work.description:
-            <h3 class="collapse">$_("About the Book")</h3>
-            $:sanitize(format(work.description))        
           $if page.description:
             <h3 class="collapse">$_("About the Edition")</h3>
             $:sanitize(format(page.description))
+          $else:
```

Comment:
We should `else` if for work and work description still; otherwise it will error in those cases

Reply:
Nope there will not will be an error as there is an `$if page.description or (work and work.description):` above this `$if page.description:`

Reply:
Ahhh, good catch! That's what I get for trying to review this too fast, :P my bad! I'll throw it on Feb for testing 👍